assets/src/adaptivity/scripting.ts

```diff
@@ -582,23 +582,23 @@ export const templatizeText = (
         strValue = stateValue.map((v) => `"${v}"`).join(', ');
       } else if (typeof stateValue === 'object') {
         strValue = JSON.stringify(stateValue);
-      } else if (typeof stateValue === 'number') {
-        const modifiedValue = formatNumber(strValue);
-        strValue = parseFloat(modifiedValue.toString());
       }
     } else {
       if (typeof stateValue === 'object' && !Array.isArray(stateValue)) {
         strValue = JSON.stringify(stateValue);
       }
     }
+    if (typeof stateValue === 'number') {
```

Comment:
this is supposed to be inside the `useFormattedText` we are only doing this for TextFlow